Code/Framework/AzFramework/AzFramework/Viewport/CameraState.cpp

```diff
@@ -10,6 +10,7 @@
 
 #include <AzCore/Math/Matrix3x4.h>
 #include <AzCore/Math/Transform.h>
+#include <AzCore/Math/Vector2.h>
```

Comment:
do you need this import?

Reply:
Good spot, I don't think I do anymore now! I can remove it (but might be a bigger change now with `Vector2i`, so might have more updates) I will fix anyway just in case :+1:

Reply:
Now removed 👍